proposal/jspss93094/README.rst

```diff
@@ -0,0 +1,103 @@
+===========================
+Proposal for Video Compression
+===========================
+
+Basic information
+=================
+
+`My GitHub repository (URL) <https://github.com/jspss93094/nsd_TermProject>`
+
+Use C++ with Python to implement the video compression, and the file type can be various.
+
+Problem to solve
+================
+
+Frequently, the videos are taken up much space if we wanted to save it, download it, or do something.
+It must took a long time to deal with a video. Even its size will be larger when the resolution is higher.
+
+Some problems I met when I want to compress the video:
+
+1. It often needed to install the software.
+2. It took a long time to compress, but the result was not better than before.
+3. The resolution became too low to watch.
+
+Perspective users
+=================
+
+The person who :
+
+1. Wanted to compress the large video.
+2. Do not want to install the software.
+3. Can not upload the file online because of the large size.
+4. Found the results of compression is not well. 
+
+System architecture
+===================
+
+It also have many modules can use, like FFMPEG. But I want to realize the process and to implement it.
+If some process in Python is better than in C++, I will embed some Python.
+
+I just use C++ to create the function to complie and execute, or the file which can be imported in Python.
+
+The function to implement : 
+
+1. Get the input.
+2. Decode the input video.
+3. Do the calculation of compression.
+4. Encode it to be the video type.
+5. Output the result and save.
+
+API description
+===============
+
+If users want to use my program in the program by themself, it can be easy to 
+
+include the file in C++. Just download my file and include it in your header.
```

Comment:
I assume you want to make your code as a header-only library.  You may spell it out in your project description.